front-end/src/components/search.js

```diff
@@ -1,12 +1,23 @@
 import React, {Component} from 'react'
 import axios from 'axios'
+import { connect } from 'react-redux'
+import action from '../ducks/action'
+const {updateName} = action
```

Comment:
one thing i would say in the future you can do:
```
import { updateName } from '../ducks/action'
```

Reply:
true, good catch.

Reply:
Will do from now on. Thank you!